models/balancer/arbitrum/balancer_arbitrum_schema.yml

```diff
@@ -76,3 +77,39 @@ models:
       - &median_price
         name: median_price
         description: 'BPT median price'
+
+  - name: balancer_arbitrum_pools_tokens_weights
+    meta:
+      blockchain: arbitrum
+      project: balancer
+      contributors: jacektrocinski
+    config:
+      tags: ['arbitrum', 'balancer', 'pools', 'tokens', 'weights']
+    description: >
+      Token weights in Balancer pools.
+    columns:
+      - &pool_id
+        name: pool_id
+        description: 'Unique encoded identifier that refers to each pool'
+        tests:
+          - not_null
+      - &token_address
+        name: token_address
+        description: 'Contract address for the token'
+      - &normalized_weight
+        name: normalized_weight
+        description: 'Weight of the token in the pool.'
+
+  - name: balancer_v2_arbitrum_pools_tokens_weights
+    meta:
+      blockchain: arbitrum
+      project: balancer
+      contributors: jacektrocinski
+    config:
+      tags: ['arbitrum', 'balancer', 'pools', 'tokens', 'weights']
+    description: >
+      Token weights in Balancer v2 pools.
+    columns:
+      - *pool_id
```

Comment:
Besides uniqueness test, you can add a seed files for these models and apply `check_seed` to the models.
The seed file will ensure the output of the models keep consistent under future changes/modifications.

Reply:
We will not be adding seed tests for now.